### PR TITLE
AD review

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

**Review by tireddy2 — APPROVED** [review]

File: draft-ietf-add-resolver-info.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ Encrypted DNS resolver:
       returned by the DoH server, or how to report a problem) for
       troubleshooting purposes.
 
-      The server MUST support the content-type 'text/html'.  The DNS
+      The resolver information server MUST support the content-type 'text/html'.  The DNS
```
Comment (tireddy2) [inline draft-ietf-add-resolver-info.md:174]: what is a "resolver information server" ?. This line needs to be re-phrased.

Comment (boucadair) [reply draft-ietf-add-resolver-info.md:174]: This is actually a term we used in OLD versions of the doc: "The client MUST validate that both the encrypted DNS server and the resolver information server are owned and managed by the same entity by establishing a TLS connection to the domain name in the URL and checking if the subjectAltName entry in the server certificate includes the name of the encrypted DNS server."

Comment (boucadair) [reply draft-ietf-add-resolver-info.md:174]: Added a new sentence "The server that exposes such information is called "resolver information server".

Comment (tireddy2) [reply draft-ietf-add-resolver-info.md:174]: Looks good.

**Review by tireddy2 — APPROVED** [review]

**Pull request merged**